### PR TITLE
fix writing rule

### PR DESCRIPTION
**Pull request by juzhiyuan**
### Summary

openresty -> OpenResty

### Issues resolved

resolve https://github.com/apache/incubator-apisix/issues/848

### CODE REVIEW

File: doc/install-dependencies.md [hunk 1]
```diff
@@ -25,17 +25,17 @@
...
 - [Debian 9 & 10](#debian-9--10)
 - [Mac OSX](#mac-osx)
-- [How to compile the Openresty](#how-to-compile-the-openresty)
+- [How to compile the OpenResty](#how-to-compile-the-openresty)
 
 CentOS 6
 ========
 
 ```shell
-# add openresty source
```
Comment (juzhiyuan) [inline doc/install-dependencies.md]: @moonming  Shoud I change `openresty` to `OpenResty`

Comment (juzhiyuan) [reply doc/install-dependencies.md]: or only `Openresty`?

File: doc/install-dependencies.md [hunk 2]
```diff
@@ -116,14 +116,14 @@ Mac OSX
...
+# install OpenResty, etcd and some compilation tools
 brew install openresty/brew/openresty etcd luarocks curl git
 
 # start etcd server with v2 protocol
 etcd --enable-v2=true &
 ```
 
-How to compile the Openresty
+How to compile the OpenResty
 =======
```
Comment (membphis) [inline doc/install-dependencies.md]: Here is the same length as the above text, better to see some.

```
How to compile the OpenResty
============================
```

File: doc/install-dependencies.md (hunk 1 above)
Comment (membphis) [reply doc/install-dependencies.md]: `OpenResty` is fine.

**Pull request merged**